app/scripts/modules/titus/migration/titusMigrationConfig.component.ts

```diff
@@ -7,6 +8,11 @@ import './titusMigrationConfig.component.less';
 
 export interface ITitusMigrationViewState extends IViewState {
   accountsLoaded: boolean;
+  isDirty: boolean;
+  originalConfig: ITitusMigrationConfig;
+  originalStringVal: string;
+  saveError: boolean;
+  saving: boolean;
```

Comment:
The compiler was complaining about missing types.  We could also mark these as optional?